/**:
  ros__parameters:
   joystick:
     remapping:
       axes:
         linear_speed: Vertical_Left_Stick
         angular_speed: Horizontal_Right_Stick
       buttons:
         slow_mode: LB
         turbo_mode: RB
